src/components/Details.tsx

```diff
@@ -75,6 +77,7 @@ const DetailItem = ({
         stringParams={tooltipParams}
         side={DETAIL_ITEM_TOOLTIP_LAYOUTS[layout]}
         withIcon={withTooltipIcon}
+        customIcon={customIcon}
```

Comment:
this is needed so i could add the "New" badge without having it be underlined -_-